pkg/syncer/controller/syncerconfigmanager.go

```diff
@@ -129,6 +132,99 @@ func (s *SyncerConfigManager) upsertNamespaceScoped(syncerConfig edgev1alpha1.Sy
 	s.syncConfigManager.upsert(edgeSyncConfig)
 }
 
+type typeFlatNamespacedObject struct {
```

Comment:
As a general rule, I do not think it is helpful in the Go programming language to put the word "type" in the name of a type.  Go is syntactically stratified, a type name can only appear where the name must name a type.  Is there a reason for an exception here?

Reply:
There isn't a significant reason... I wanted to make this type name a private scope in this module. To do that, I defined its name as starting small letter. The reason why I add a prefix `type` rather than just naming `flatNamespacedObject` is that I just wanted to use `typeFlatNamespacedObject` as a local variable name.

Reply:
I am not sure I understand that answer.  I see `typeFlatNamespacedObject` used as a package-scoped type name, not a more narrowly-scoped variable name.  I do not see `flatNamespacedObject` used at all.

Reply:
> I do not see flatNamespacedObject used at all.

I thought I used `flatNamespacedObject` as a name of a local variable for `typeFlatNamespacedObject`. There is no reason to put `type` in the name of type. I will remove it.